pandas/tests/test_base.py

```diff
@@ -38,7 +38,6 @@ def test_tricky_container(self):
             pytest.skip('Need unicode_container to test with this')
         repr(self.unicode_container)
         str(self.unicode_container)
-        bytes(self.unicode_container)
```

Comment:
This raises in the FrozenList case, but not the FrozenNDArray case. So I move the mest to test_frozen.py and have seperate tests for FrozenList and FrozenNDArray.